### PR TITLE
fix: Add multilanguage support

### LINKED ISSUE #2001: Multilanguage support not working properly
**Describe the bug**
When I select a different language from the settings activity for my app, all the strings are not getting translated, though they have their string value defined in the strings file. But if I change the default language of the mobile and open the app, it is working fine.

**Expected Behaviour**
App should be able to display itself in the language  selected from the settings activity of the app.

**Steps to reproduce it**
Need to change the code in setLanguage function of ChatSettingsFragment

**Screenshots of the issue**

ezgif com-video-to-gif (1)

**Smartphone Info:**

|               | |
|---------------|-|
|Device         | Moto G5 Plus|
|Android Version|  Oreo 8.1   |

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by atm1504**
Fixes #2001 

Changes:
* Added the setLocate function in ChatSettingsFragment.kt. 
This function is changing the language of the app

Screenshots for the change: 
multiLang

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt [hunk 1]
```diff
@@ -267,6 +276,13 @@ class ChatSettingsFragment : PreferenceFragmentCompat(), ISettingsView {
         }
     }
 
+    private fun setLocate(Lang: String) {
```
Comment (batbrain7) [inline app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: should it not be setLocale

Comment (atm1504) [reply app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: SIr, here I have used a custom function setLocate. setLocale object has been created inside this function

Comment (batbrain7) [reply app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: use a more meaningful name instead of setLocate

Comment (atm1504) [reply app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: @batbrain7 please check it now

**Comment by atm1504** [comment]
@batbrain7 @iamareebjamal please check it

**Comment by atm1504** [comment]
@iamareebjamal @batbrain7 please help this to resolve it. I have pulled it and applied the spotlessApply. But there is no change in file after applying that. It shows that my branch is all right. Also it contains all the latest commits of development branch.

**Comment by atm1504** [comment]
@iamareebjamal please review it

**Comment by atm1504** [comment]
@iamareebjamal please review it.

File: app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt [hunk 2]
```diff
@@ -273,6 +275,13 @@ class ChatSettingsFragment : PreferenceFragmentCompat(), ISettingsView {
                 val index = querylanguage.findIndexOfValue(PrefManager.getString(Constant.LANGUAGE, Constant.DEFAULT))
                 querylanguage.setValueIndex(index)
                 querylanguage.summary = querylanguage.entries[index]
+                if (index == 1) setLocalLanguage("de")
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: PrefManager.getString(Constant.LANGUAGE, Constant.DEFAULT)

When you have this, why searching from index?

Comment (atm1504) [reply app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: I have changed it. Please check it now

**Comment by atm1504** [comment]
@iamareebjamal please merge it.

**Pull request merged**